docs/developing.md

```diff
@@ -108,7 +108,9 @@ go install github.com/cloudflare/cfssl/cmd/cfssl
 You can now build your local Hive source as a container, push to the local registry, and deploy Hive. Because we are not running on OpenShift we must also create a secret with certificates for the hiveadmission webhooks.
 
 ```bash
-IMG=172.17.0.1:5000/hive:latest make docker-dev-push
+make build
```

Comment:
This was a dependency of docker-dev-push, is it now not really feasible to continue that with image-hive-dev? I can live with it if not, and from the looks of how this works now it feels non-trivial to add.

Reply:
I can put the `docker-dev-push` target back.